test/jdk/java/awt/PopupMenu/PopupMenuVisuals.java

```diff
@@ -81,7 +80,7 @@ private static Frame createTestUI() {
 
         //Get things going.  Request focus, set size, et cetera
```

Comment:
minor formatting change
```suggestion
        // Get things going. Request focus, set size, et cetera
```

Reply:
Fixed.